images/win/scripts/Installers/Install-JavaTools.ps1

```diff
@@ -33,16 +33,16 @@ foreach ($pathSegment in $pathSegments)
     }
 }
 
-$java7Installs = Get-ChildItem -Path 'C:\Program Files\Java' -Filter '*azure-jdk*7*' | Sort-Object -Property Name -Descending | Select-Object -First 1
+$java7Installs = Get-ChildItem -Path 'C:\Program Files\Java' -Filter '*azure-jdk_7*' | Sort-Object -Property Name -Descending | Select-Object -First 1
```

Comment:
hopefully they won't release version 70 any time soon :-)

Reply:
Changed * to .*